server/notification-providers/mobivatesms.js

```diff
@@ -0,0 +1,47 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class MobivateSMS extends NotificationProvider {
+    name = "MobivateSMS";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+        const url = "https://vortex.mobivatebulksms.com/send/batch";
+
+        try {
+            // smspartner does not support non ascii characters and only a maximum 639 characters
+            let cleanMsg = msg.replace(/[^\x00-\x7F]/g, "").substring(0, 639);
+
+            let data = {
+                "originator": notification.mobivateOriginator.substring(0, 15),
+                "recipients": notification.mobivateRecipients.split(",").map(n => n.replace(/[^0-9]/g, "")).filter(n => n.length >= 9).map(recipient => ({recipient})),
```

Comment:
Filtering a recipient away is not very user obvious.

Could we either throw an error in such a case (obvious because the test button does not work) or add validation to the frontend?